php-frontend/src/main/java/org/sonar/php/cfg/ControlFlowGraphBuilder.java

```diff
@@ -106,24 +124,85 @@ private PhpCfgBlock build(Tree tree, PhpCfgBlock currentBlock) {
     }
   }
 
+  private PhpCfgBlock buildBreakStatement(BreakStatementTree tree) {
+    PhpCfgBlock newBlock = createSimpleBlock(getBreakable(tree.argument(), tree).breakTarget);
+    newBlock.addElement(tree);
+    return newBlock;
+  }
+
+  private PhpCfgBlock buildContinueStatement(ContinueStatementTree tree) {
+    PhpCfgBlock newBlock = createSimpleBlock(getBreakable(tree.argument(), tree).continueTarget);
+    newBlock.addElement(tree);
+    return newBlock;
+  }
+
+  private Breakable getBreakable(@Nullable ExpressionTree argument, StatementTree jumpStmp) {
+    try {
+      if (argument != null) {
+        if (!argument.is(Kind.NUMERIC_LITERAL)) {
+          throw exception(jumpStmp);
+        }
+        Iterator<Breakable> breakableIterator = breakables.iterator();
+        Breakable breakable = breakableIterator.next();
+        int breakLevels = Integer.parseInt(((LiteralTree) argument).value());
```

Comment:
lines 147-150 can be moved in a `parseBreakLevel` method, would improve readability